core/cmd/vrf_keys_commands.go

```diff
@@ -0,0 +1,277 @@
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	"github.com/smartcontractkit/chainlink/core/services/signatures/secp256k1"
+	"github.com/smartcontractkit/chainlink/core/web/presenters"
+	"go.uber.org/multierr"
+
+	"github.com/pkg/errors"
+	"github.com/smartcontractkit/chainlink/core/utils"
+	"github.com/urfave/cli"
+)
+
+type VRFKeyPresenter struct {
+	JAID // Include this to overwrite the presenter JAID so it can correctly render the ID in JSON
+	presenters.VRFKeyResource
+}
+
+func (p VRFKeyPresenter) FriendlyDeletedAt() string {
+	if p.DeletedAt != nil {
+		return p.DeletedAt.String()
+	}
+	return "\n"
```

Comment:
Why does this return a newline?

Reply:
Without it the output looks like this 
```
connorstein@Connors-MBP ~/go/src/github.com/smartcontractkit/chainlink (feature/ch11205) $ ./chainlink keys vrf list                       (43 results) [829/1927]
🔑 VRF Keys
------------------------------------------------------------------------------------------------------------------------------------------------
Compressed:   0xb1fa7a53f0a03f25f708da3b00188ad0546665d50025255f06eebe460459825b01
Uncompressed: 0xb1fa7a53f0a03f25f708da3b00188ad0546665d50025255f06eebe460459825b4cce3a07d9a0491bb8b9a06384b41bdd48b57d226743af227570e514b4e1ce65
Hash:         0xd47a5deec6b049b4ad671278568b7e620ec5abb2c99aa8b597408c4d395f7dcc
Created:      2021-06-15 22:42:56.390954 -0400 EDT
Updated:      2021-06-15 22:42:56.390954 -0400 EDT
Deleted:      connorstein@Connors-MBP ~/go/src/github.com/smartcontractkit/chainlink (feature/ch11205) $
```

Reply:
It doesn't really make sense to put formatting code inside the return for a getter. 

I think it would be better suited to the render function event if you have to put it in twice. I've added a suggestion below for it